engine/class_modules/sc_mage.cpp

```diff
@@ -2464,6 +2464,24 @@ struct mage_spell_t : public spell_t
 
     p()->state.trigger_glorious_incandescence = false;
   }
+
+  void trigger_intuition( bool BLP_exclude_initial )
+  {
+    if ( !p()->talents.intuition.ok() )
+      return;
+
+    const int BLP_threshold { 11 };
+    // If an action and it's direct after-effect can increment intuition's BLP,
+    // exclude the second incoming incrementation if it already gained intuition from its previous associated action
```

Comment:
You're trying to explain what `BLP_exclude_initial` is for, yes? If so, it would be better to have it in front of the method rater than somewhere in the middle.